dashboard/config/levels/custom/dance/dance_ai_try_key_press.level

```diff
@@ -6,6 +6,7 @@
   "level_num": "custom",
   "user_id": 18605,
   "properties": {
+    "uses_preview": "true",
```

Comment:
As a note for later, should we add a levelbuilder toggle for this value, or do we always want to change this via PR?

Reply:
Since preview is always 'on' in Sprite Lab and Poetry Lab, I wonder if we should ask product if this is a feature they want for legacy progressions? I think when we first added this feature, they didn't want any changes in legacy. But it seems like we've since made changes that have impacted legacy (remixing the project-backed freeplay level, hints/validation). Maybe we can just remove the property then?

If product/curric say they don't want to turn on feature for legacy, then yes, I think we should add as a lb toggle.